leger/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+from datetime import date
+from django.db.models import Sum
+
+
+class User(models.Model):
```

Comment:
Can you subclass this off of from django.contrib.auth.models.AbstractUser instead of models.Model? :) That'll bring in all the extra user goodies. :)